examples/policies.py

```diff
@@ -70,3 +70,11 @@
 vms = policy.get_vms()
 print(policy.data)
 print(vms)
+
+print("\n\n create policy")
+policy_name = "test_policy"
+policy = policies.create(policy_name)
```

Comment:
it's good to have timestamped names but as we will not be having many policies like backups, don't you think we should have a meaningful name for the policies to explain what exactly the policies do. e.g. "default_policy", "every_hour"

Reply:
I agree with your thoughts on having the proper name, the reason 'test_policy' was to indicate a temporary name, however since this is part of example.py we can rename this policy as "example_policy" to indicate to user that this just example and he can choose any name he wants, we can also write a comment on top to be more explicit.
Please let me know your views so that we can make necessary changes.

Reply:
Good, go ahead with a valid name and leave a comment if required.

Reply:
Updated the name to some valid name as "fixed_frequency_retention_policy"
Since we are also working add rules to policy we can continue using this name for rule creation.
Rule create code will be added in the same block to complete the complete the policy and rule creation deletion.